# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pcmcia-cs/tools-only.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- pcmcia-cs-3.2.8/Makefile	2003-05-16 07:59:44.000000000 +0200
+++ pcmcia-cs-3.2.8-fixed/Makefile	2004-10-16 23:29:13.054710728 +0200
@@ -6,10 +6,10 @@
 include config.mk
 endif
 
-ALL  = modules clients wireless cardmgr flash debug-tools man etc
+ALL  = clients wireless cardmgr flash debug-tools man etc
 DIRS = cardmgr flash debug-tools man etc
 ifndef CONFIG_PCMCIA
-DIRS := modules clients wireless $(DIRS)
+DIRS := $(DIRS)
 endif
 
 help:
